packages/api/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
-  "extends": "@agreeto/tsconfig/node16.json",
-  "compilerOptions": {
-    "declaration": false,
-    "declarationMap": false
-  },
-  "include": ["src", "**/*.ts", "transformer.ts"],
-  "exclude": ["node_modules"]
+  "extends": "@agreeto/tsconfig/test.json",
```

Comment:
reminder to look into the tsconfigs. not sure if they were the problem or if it was something else